### PR TITLE
Add a page talking about the 32-bit deprecation

### PR DESCRIPTION
**Pull request by dscho**
This is needed to implement [this suggestion](https://github.com/git-for-windows/build-extra/pull/482#issuecomment-1439154393).

### CODE REVIEW

**Comment by dscho** [comment]
The current page (after running `grunt`) looks like this:

image

File: 32-bit.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# 32-bit support of Git for Windows
+
+While Git for Windows v1.x was only ever offered as 32-bit installer (i.e. targeting the i686 CPU architecture), as of Git for Windows v2.x (which happened mid-2015), there have been two variants: the 32-bit and the 64-bit (x86\_64) one.
+
+However, due to Git for Windows' reliance on the MSYS2 runtime, which dropped 32-bit support as of version 3.4.x, Git for Windows will start to phase out 32-bit support after Git for Windows v2.40.x.
```
Comment (mjcheetham) [inline 32-bit.md:5]: Maybe a link to the MSYS runtime 3.4.x release notes/docs about dropping 32-bit support? For those that care to dive deeper?

**Review by mjcheetham — APPROVED** [review]
Great! Just one optional suggestion of adding a link to MSYS2's 32-bit announcement/drop, if you think people would like it.

File: 32-bit.md (hunk 1 above)
Comment (dscho) [reply 32-bit.md]: Done! I also reworded the introduction in places where I found it klunky.

**Pull request merged**